app/Http/Controllers/ApiController.php

```diff
@@ -58,5 +61,28 @@ protected function permissionOneOf(Permission ...$permissions): ControllerMiddle
         );
     }
 
+    protected function getRequestIn(): array
+    {
+        if (!isset($this->requestIn)) {
+            $in = $this->request->validate(['in' => 'nullable|string|lowercase'])['in'] ?? null;
+            if ($in) {
+                $inArr = explode(',', $in);
+                Validator::validate(['in' => $inArr], ['in.*' => 'bail|required|string|uuid']);
```

Comment:
ja ma być uuid to chyba nie musisz dodatkowo sprawdzać, że musi być stringiem

Reply:
a no w sumie wywalę, bo rzeczywiście chyba nawet nie da się tutaj przekazać nie-stringa

to bail tez wywalę